bin/node/runtime/pangolin/src/lib.rs

```diff
@@ -1094,6 +1094,7 @@ impl darwinia_evm::Trait for Runtime {
 		darwinia_evm_precompile_simple::Sha256,
 		darwinia_evm_precompile_simple::Ripemd160,
 		darwinia_evm_precompile_simple::Identity,
+		darwinia_evm_precompile_withdraw::WithDraw<Self>,
```

Comment:
Native contract may start at address `0x400` for compatibling evm.

Reply:
What you said is indeed a problem, in order to maintain maximum compatibility with ethereum, a minimum of 0x1-0x20 address space should be set aside. But in the dvm implementation, the precompile contract address is determined by the order set in runtime.

1. https://github.com/darwinia-network/darwinia-common/blob/master/primitives/evm/src/precompile.rs#L64
2. https://github.com/darwinia-network/darwinia-common/blob/master/bin/node/runtime/pangolin/src/lib.rs#L1092

Perhaps we can add some mock precompile contracts in the runtime to occupy this address space.

Reply:
How about set some precompile placeholder instances here, which could be just as empty now, can be replaced later using runtime upgrade

Reply:
I think this approach works and I will create a separate issue on this problem.